### DISCUSSION TITLE
The Default Email & Password for Dashboard

### DISCUSSION THREAD

**Discussion by MichaelNaghavipour** [saleor/saleor#6955] [Q&A]
I have installed saleor source code locally. 
What is the default Email & Password for Dashboard?

**Comment by karolkielecki** [comment]
Hey Michael, 

You need to create a superuser account. You can find more information here: https://docs.saleor.io/docs/next/developer/installation#installation-using-docker-compose

Good luck!

**Reply by itwasmee** [reply]
your link is dead it seems

**Comment by itwasmee** [comment]
you have to create a super user through Django, it may be different for every version but you can try `manage.py createsuperuser`. If you used docker, just execute the command inside the container.
